### PR TITLE
Floating headers in HTML output

### PR DESCRIPTION
**Pull request by mattparlane**
This is a new version of #2657, including a changelog update.

### CODE REVIEW

**Comment by mattparlane** [comment]
Sorry, I had to create a new branch and rebase in order to create a changelog entry. Let me know if I did it wrong...  :D

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 ### New features
 
+* [#2657](https://github.com/bbatsov/rubocop/pull/2657): Floating headers in HTML output. ([@mattparlane][])
```
Comment (bbatsov) [inline CHANGELOG.md]: You forgot add yourself to the bottom of this file.

**Comment by mattparlane** [comment]
There you go... :D

**Comment by bbatsov** [comment]
:+1:

**Pull request merged**